src/components/Sort/__mocks__/mockSortExample.js

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+import { Sort, Toolbar } from '../../../index';
+
+const bindMethods = (context, methods) => {
```

Comment:
should just be able to:
import { bindMethods } from '../../../common/helpers';

it seems to work here...